### ISSUE TITLE
When creating a developer scratch org, the community 'Report Builder' feature wont be available

### ISSUE THREAD

**Issue by vviotto-masti** [forcedotcom/cli#396]
### Summary

After the spring 20 release, [a new feature](https://releasenotes.docs.salesforce.com/en-us/spring20/release-notes/rn_networks_report_builder.htm) became available for communities, this feature allows community members to create reports inside the community.
If you create a new scratch org with force:org:create using a definition file with the org edition set to 'Developer' this feature wont be available.
This is not the case if the edition is set to 'Enterprise'.

### Steps To Reproduce:

1.  Create a new scratch org using the following definition file
```{
  "orgName": "Demo company",
  "edition": "Developer",
  "features": [
    "API",
    "Communities",
    "ConnectedAppToolingAPI",
    "DebugApex",
    "EinsteinAnalyticsPlus",
    "Interaction",
    "MarketingUser",
    "MultiCurrency",
    "PardotScFeaturesCampaignInfluence",
    "PersonAccounts",
    "ProcessBuilder",
    "RecordTypes",
    "SalesforceIdentityForCommunities",
    "Sites",
    "Workflow",
    "WorkflowFlowActionFeature"
  ],
  "settings": {
    "omniChannelSettings": {
      "enableOmniChannel": true
    },
    "quoteSettings": {
      "enableQuote": true
    },
    "languageSettings": {
      "enableTranslationWorkbench": true
    },
    "lightningExperienceSettings": {
      "enableS1DesktopEnabled": true
    },
    "communitiesSettings": {
      "enableNetworksEnabled": true,
      "enableNetPortalUserReportOpts": true
    },
    "analyticsSettings": {
      "enableAnalyticsSharingEnable": true,
      "enableLightningReportBuilder": true,
      "enableMassEnableReportBuilder": true
    }
  }
}
```
1.  open the scratch org and go to Setup -> All communities and create a new community with the 'Partner Central' or the 'Customer Account Portal' template
1. After creation, in the workspace page select 'Builder'
1. At the top left of the page there is a button to change the page, click there and type 'Report'
1. Change the edition from 'Developer' to 'Enterprise' and repeat steps 1-3

### Expected result

There should be a page called 'Report Builder', and the 'New Report' button should display inside the the 'Report List' page.

### Actual result

No report builder page is created and the button is not visible

### Additional information

This is not a problem with the Developer edition because the feature should be available for developer orgs and I've confirmed that if you repeat the steps in a Developer Sandbox the feature will be available correctly, so it should be a problem with the scratch org creation.

**SFDX CLI Version**: 7.54.4

**SFDX plugin Version**
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.3 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.9 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.7 (core)
analytics 1.7.1 (core)
generator 1.1.2 (core)
salesforcedx 48.10.1 (core)
├─ salesforcedx-templates 48.8.0 (core)
├─ salesforce-alm 48.10.0 (core)
└─ @salesforce/sfdx-plugin-lwc-test 0.1.5 (core)

**OS and version**: Win 10

**Comment by vviotto-masti** [comment]
After further investigation, the problem seems to be intermittent, because after testing more than 10 times, I just created a new scratch org using the exact same steps with the exact same definition file and the feature is available.
I've noticed that this also happens with a  community setting called 'enableGuestRecordReassignOrgPref', sometime its available, sometimes its not.

Our original problem was because we already have source files from a community that is using the new experience bundle, and this community has the 'createreport' route, this source was giving errors when pushing to a new scratch org because the 'Report builder' feature was not available.

It seems to me that some scratch orgs are being created with some features disabled, but I'm not able to pinpoint what the problem might be.
